.travis.yml

```diff
@@ -49,6 +53,24 @@ install:
 
   # install TensorFlow (CPU version).
   - pip install tensorflow
+  
+  # install cntk
+  - if [[ "$TRAVIS_PYTHON_VERSION" == "2.7" ]]; then
+      wget http://cntk.ai/PythonWheel/ForKeras/CPUOnly/cntk-2.0rc3-cp27-cp27mu-linux_x86_64.whl;
+      pip install cntk-2.0rc3-cp27-cp27mu-linux_x86_64.whl;
```

Comment:
If you're not validating the checksum, why not just `pip install http://...` directly?